presto-docs/src/main/sphinx/release/release-340.md

```diff
@@ -0,0 +1,60 @@
+# Release 340 (XX Aug 2020)
+
+## General Changes
+
+* Add support for query parameters in `LIMIT`, `OFFSET` and `FETCH FIRST` clauses. ({issue}`4529`, {issue}`4601`)
+* Add {func}`bitwise_left_shift`, {func}`bitwise_right_shift` and {func}`bitwise_right_shift_arithmetic`. ({issue}`772`)
+* Improve dynamic partition pruning for short queries. ({issue}`4388`}
```

Comment:
This is quite vague. What was improved? What kind of improvements user might expect? Shorter latency, lower memory footprint? What does it mean short query?

FYI @sopel39